feat: create new blog post (#2876)
agrc/gis.utah.gov: src/content/blog/2024-10-17-the-data-process-that-powers-elections-in-utah.mdx

---
title: The Data Process That Powers Elections in Utah
author: Miriam Seely
description: >-
  As you prepare for election day this year, learn more about how geospatial
  data impacts where, when, and how you vote in Utah.
date: '2024-10-17T16:31:16.697Z'
category: UGRC
cover_image: /src/images/pillar-blog/default-social-card.png
cover_image_alt: ugrc social card
---

import Contacts from '@components/page/Contacts.astro';

With the cool winds blowing in and the birds flying south, fall is officially upon us. That means a few things: warm sweaters, yellow aspen leaves, and everyone's favorite method of getting a free sticker: voting! Election Day will be on Tuesday, November 5th, 2024. If you aren't yet registered, do it now on [vote.utah.gov](http://vote.utah.gov) and vote by mail or in person! Did you know that where you vote and who's on your ballot are determined by geospatial data maintained by UGRC and local government authorities?

> UGRC might not be able to make the beautiful autumn hues of red, orange, and yellow stay longer than their natural time, but we are charged with safekeeping the data that makes elections in Utah possible. An example of these data are the [digital polygon representations](src\pages\products\sgid\political\voter-precincts.astro) of voting precinct boundaries. Maintaining these boundaries requires collaboration, organization, and a great attention to detail. Part of [our mission](/about/) as geospatial leaders in Utah is to “ensure a high level of coordination among Utah GIS users and effective, efficient use of GIS resources.” And boy oh boy, does making elections happen demand a lot of coordination. So, let's take a moment and get to know what's involved with keeping the data side of democracy alive.

Reviewer: ```suggestion
UGRC might not be able to make the beautiful autumn hues of red, orange, and yellow stay longer than their natural time, but we are charged with safekeeping the data that makes elections in Utah possible. An example of these data are the [digital polygon representations](/products/sgid/political/voter-precincts/) of voting precinct boundaries. Maintaining these boundaries requires collaboration, organization, and a great attention to detail. Part of [our mission](/about/) as geospatial leaders in Utah is to “ensure a high level of coordination among Utah GIS users and effective, efficient use of GIS resources.” And boy oh boy, does making elections happen demand a lot of coordination. So, let's take a moment and get to know what's involved with keeping the data side of democracy alive.
```

relative links are URL links and not file system links. You don't need src\pages and astro extensions.

> UGRC might not be able to make the beautiful autumn hues of red, orange, and yellow stay longer than their natural time, but we are charged with safekeeping the data that makes elections in Utah possible. An example of these data are the [digital polygon representations](https://gis.utah.gov/products/sgid/political/voter-precincts/) of voting precinct boundaries. Maintaining these boundaries requires collaboration, organization, and a great attention to detail. Part of [our mission](https://gis.utah.gov/about/) as geospatial leaders in Utah is to “ensure a high level of coordination among Utah GIS users and effective, efficient use of GIS resources.” And boy oh boy, does making elections happen demand a lot of coordination. So, let's take a moment and get to know what's involved with keeping the data side of democracy alive.

Reviewer 2: ```suggestion
UGRC might not be able to make the beautiful autumn hues stay longer than nature dictates, but we are charged with safekeeping the data that makes elections in Utah possible. One example is the [digital polygon representations](https://gis.utah.gov/products/sgid/political/voter-precincts/) of voting precinct boundaries, which is part of [our mission](https://gis.utah.gov/about/) as geospatial leaders in Utah to “ensure a high level of coordination among Utah GIS users and effective, efficient use of GIS resources.” And boy oh boy do elections demand a lot of coordination. So, let's take a moment and get to know what's involved with keeping the data side of democracy alive.
```

> All 29 counties in the state of Utah employ public servants who manage surveying, recording, and verifying their county boundaries and voting precincts. These individuals, often County Recorders and County Clerks, submit their data to UGRC many months before election day. Every county sends in slightly different datasets according to the needs and circumstances of that county. UGRC adjusts each incoming dataset to fit a standardized schema and ensure accuracy of the information. The completed boundaries are used in elections at all levels, including city, county, state, and federal elections.

Reviewer 2: ```suggestion
All 29 counties in the state have public servants, often the County Recorders and Clerks, in charge of surveying, recording, and verifying their county boundaries and voting precincts. Each county submits these datasets many months before election day. Their schemas often differ slightly to meet each county's unique needs, so UGRC adjusts them to a standardized schema, ensures their spatial accuracy, and adds them to the statewide datasets. The completed boundaries are then used in elections at all levels—city, county, state, and federal.
```

It could also be nice to have a paragraph before this explaining exactly what a precinct is and how it's used in elections. Then this paragraph explains how they flow from counties to us, and the next can explain the consequences of errors (double count, wrong count, not counted) or accurate datasets (voters can have confidence in the election).

Reviewer 3: I like that idea. I'll add a bit more about what voting precincts are.

Without these geographic boundaries, votes could be double-counted, counted in the wrong county, or not counted at all. That's why each dataset must pass a rigorous screening process to ensure accuracy. Once ballots have been sent out prior to election day, these boundaries cannot be altered or adjusted in any way until after the election results have been certified. After the election, voters can access election outcome maps available on Utah's official [election results website](https://electionresults.utah.gov/results/public/utah/elections/PrimaryCD2Recount2024).

> The State of Utah's use of GIS in election operations is part of a wider effort to include geospatial technology in democratic processes in the United States. According to a [2022 report](https://nsgic.memberclicks.net/assets/docs/GEE/2022_October_StateElectionDirectorReport_final.pdf) prepared by the National States Geographic Information Council, 90% of states in the US report that their voter registration system will incorporate GIS within the next five years. This is great news for voters; it means greater accuracy, transparency, and clarity for US elections.

Reviewer 2: ```suggestion
The State of Utah's use of GIS in election operations is part of a wider effort to include geospatial technology in democratic processes in the United States. According to a [2022 report](https://nsgic.memberclicks.net/assets/docs/GEE/2022_October_StateElectionDirectorReport_final.pdf) prepared by the National States Geographic Information Council, 90% of states in the US report that their voter registration system will incorporate GIS within the next five years. This is great news for voters: it means greater accuracy, transparency, and clarity for US elections.
```

> You can learn more about how UGRC is using data to improve life in Utah on our [blog](https://gis.utah.gov/blog/). For questions or concerns regarding election data and ballot areas in Utah, please reach out to <Contacts inline subject={frontmatter.title} contactKey='boundaries' />.

Reviewer: Anytime you want to add a https://gis.utah.gov link to a blog post please remove the `https://gis.utah.gov` part of the url to make it relative please.
